lte/gateway/c/oai/tasks/amf/amf_config.c

```diff
@@ -117,7 +125,10 @@ void ngap_config_init(ngap_config_t* ngap_conf) {
 **                                                                        **
 ***************************************************************************/
 void amf_config_init(amf_config_t* config) {
+  memset(config, 0, sizeof(*config));
+
   pthread_rwlock_init(&config->rw_lock, NULL);
+
   config->max_gnbs                       = 2;
```

Comment:
do we need to source max_gnbs, max_UEs as hard coded values like this? i think we need to discuss these.

Reply:
Suresh, this is used for hashtable sizing (i.e., num of buckets), not as a limit on connected eNBs or gNBs.